src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -308,16 +308,16 @@ public function getWriteInfo(string $class, string $property, array $context = [
             $mutator->setRemoverInfo(new PropertyWriteInfo(PropertyWriteInfo::TYPE_METHOD, $removerAccessName, $this->getWriteVisiblityForMethod($removerMethod), $removerMethod->isStatic()));
 
             return $mutator;
-        } else {
-            $errors = array_merge($errors, $adderAndRemoverErrors);
         }
 
+        array_push($errors, ...$adderAndRemoverErrors);
```

Comment:
Mostly motivated by the usage of array_merge in loops, which are more greedy for memory consumption, but updated everywhere so it's consistent, and we don't really need a merge anyway (keys don't matter)

Reply:
Unfortunately, in php <7.3, if `$adderAndRemoverErrors` is empty: 

```
array_push() expects at least 2 parameters, 1 given
```